### ISSUE TITLE
Setting channel ID

### ISSUE THREAD

**Issue by TooMuchRAM** [PopTudor/Java-fcm#4]
Since Android O, there's an option to set a channel id in a Firebase notification. Is there any way to set this in this library?
See [the Firebase docs](https://firebase.google.com/docs/cloud-messaging/http-server-ref#table2)

**Comment by tudor-pop** [comment]
Unfortunately I haven't added that functionality since there aren't a lot of users of the lib. If you desire to add it, it should be pretty simple by just adding a new field(channelId) to [Notification class](https://github.com/PopTudor/Java-fcm/blob/master/src/main/java/org/pixsee/fcm/Notification.java) and a new method to the builder. Just looked over [docs](https://firebase.google.com/docs/cloud-messaging/http-server-ref#notification-payload-support) and the field is inside Notification so I believe it is only useful when sending foreground notifications. Channels for background notifications can just be sent inside the "data" field because(if I remember correctly) channels are created from within the app and not passed from backend. How I think we've done this on a project is that we were sending a notification_type and based on that, we were routing to different channels inside the app. If you feel adventurous and are sure that we need that field, please create a merge-request and I will gladly accept it. Try testing it out from postman or the console before starting to fork the lib(just so you don't waste time)

**Comment by tudor-pop** [comment]
[foreground vs background notifications](https://firebase.google.com/docs/cloud-messaging/android/receive)

**Comment by TooMuchRAM** [comment]
I'm still pretty new to Git, so making a PR is going to take some time for me, but I'll do what I can.

**Comment by TooMuchRAM** [comment]
As it seems to me, I need more permission to be able to create a new branch and thus create a new PR. I still don't really know how this works, but that's what GitHub Desktop told me. If there's any other way to create a PR, please explain me.

**Comment by tudor-pop** [comment]
Just make sure you push your changes and after that come back to github and click "new pull request". [some docs](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) or [video](https://www.youtube.com/watch?v=rgbCcBNZcdQ)

**Comment by TooMuchRAM** [comment]
I managed to create a PR: https://github.com/PopTudor/Java-fcm/pull/5

**Comment by tudor-pop** [comment]
awesome! thank you for the contribution!
